src/ExtensionsSample/Samples/ApiHubTableSamples.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.ApiHub.Sdk.Table;
+using Microsoft.Azure.WebJobs;
+using Newtonsoft.Json.Linq;
+
+namespace ExtensionsSample.Samples
+{
+    // To use the ApiHubTableSamples:
+    // 1. Add an AzureWebJobsSql app setting or environnment variable for your ApiHub SQL connection. 
+    //    The format should be: Endpoint={endpoint};Scheme={scheme};AccessToken={accesstoken}
+    // 2. Call config.UseApiHub() in Program.cs
+    // 3. Add typeof(ApiHubTableSamples) to the SamplesTypeLocator in Program.cs
+    // 4. Create table1 in your SQL database:
+    //      CREATE TABLE table1
+    //      (
+    //          c1 int NOT NULL,
+    //          c2 nvarchar(10) NULL
+    //          CONSTRAINT PK_table1 PRIMARY KEY(c1)
+    //      )
+    public static class ApiHubTableSamples
+    {
+        public static void BindToTableClient(
+            [ApiHubTable("AzureWebJobsSql")]
```

Comment:
The problem is that there could be a bunch of ApiHub tabular connector types (Sql, GoogleSheets, Excel, etc) and a few connections for each connector type, all used in the same function app. Maybe we could invent some notion of default but I am not sure how useful that will be.